install/kubernetes/templates/istio-ingress.yaml.tmpl

```diff
@@ -42,7 +42,7 @@ spec:
       serviceAccountName: istio-ingress-service-account
       containers:
       - name: istio-ingress
-        image: {PROXY_HUB}/proxy:{PROXY_TAG}
+        image: {PROXY_HUB}/proxy_debug:{PROXY_TAG}
```

Comment:
why?

Reply:
because I can only get testing passing why proxy_debug is used in ingress, for reasons I don't understand yet but my ingress log had load cert issue when proxy image is used.  I need to open an separate issue for that for further investigation.

Reply:
oh that's very strange. thanks for noticing/filling the issue